iphone/Classes/TiLottieVectorView.m

```diff
@@ -10,13 +10,29 @@
 
 @implementation TiLottieVectorView
 
-- (LAAnimationView *)animationView
+- (LOTAnimationView *)animationView
 {
     if (animationView == nil) {
         id resource = [[self proxy] valueForKey:@"resource"];
+        id mode = [[self proxy] valueForKey:@"contentMode"];
         ENSURE_TYPE(resource, NSString);
+        ENSURE_TYPE(mode, NSString);
+        
+        animationView = [LOTAnimationView animationNamed:resource];
+        
+        if ([mode isEqual: @"aspectFit"]) {
```

Comment:
I'd prefer to use constants here, so we don't have to use string-compares. I can add the required changes tomorrow, are you ok with that?